diff --git a/Sources/CNIOBoringSSL/crypto/evp/print.cc b/Sources/CNIOBoringSSL/crypto/evp/print.cc
index 89ceb32..5e6fb2f 100644
--- a/Sources/CNIOBoringSSL/crypto/evp/print.cc
+++ b/Sources/CNIOBoringSSL/crypto/evp/print.cc
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.

+#include <inttypes.h>
+
 #include <CNIOBoringSSL_evp.h>
 
 #include <CNIOBoringSSL_bio.h>
